test/unit/data-types/enum.test.ts

```diff
@@ -0,0 +1,25 @@
+import { DataTypes } from '@sequelize/core';
+
+describe('DataTypes.ENUM', () => {
+  describe('setting values for enums', () => {
+    it('should not throw if `values` is a readonly array', () => {
```

Comment:
It's typing, it doesn't throw anything

```suggestion
    it('accepts readonly arrays for `options.values`', () => {
```